h/templates/panels/navbar.html.jinja2

```diff
@@ -44,7 +44,7 @@
             presses Enter in the search bar input, the browser will act as if
             the user had clicked one of the form's other submit elements
             (e.g. the leave group button). #}
-        <input type="submit" class="u-hidden">
```

Comment:
"u-hidden" isn't used anywhere anymore but it seems useful to have / inevitable that we will need it one day so I left it in the CSS